lib/utils.js

```diff
@@ -1,5 +1,8 @@
 var _ = require('underscore');
 
+var DEFAULT_PREFIXES;
+var DEFAULT_DEFAULT_PREFIX = 'logstash-';
```

Comment:
I considered it! I like `DEFAULT_DEFAULT_PREFIX`, though. Kinda rolls off the tongue, you know?